geth/node/rpc_test.go

```diff
@@ -33,7 +57,121 @@ func (s *RPCTestSuite) SetupTest() {
 	nodeManager := node.NewNodeManager()
 	require.NotNil(nodeManager)
 
+	acctman := node.NewAccountManager(nodeManager)
+	require.NotNil(acctman)
+
+	txQueueManager := node.NewTxQueueManager(nodeManager, acctman)
+	require.NotNil(txQueueManager)
+
+	policy := jail.NewExecutionPolicy(nodeManager, acctman, txQueueManager)
+	require.NotNil(policy)
+
+	s.Policy = policy
+	s.Account = acctman
 	s.NodeManager = nodeManager
+	s.TxQueueManager = txQueueManager
+}
+
+func (s *RPCTestSuite) TestRPCSendTransaction() {
```

Comment:
By moving I also meant to make it similar to `TestCallRPC` :P So, I would get rid of all these implementation details like `txQueueManager`, `ExecutionPolicy` and just follow the pattern:
1. Get `rpcClient`,
2. Generate config with upstream enabled,
3. Start node,
4. Call `rpcClient.Call(inputJSON)`.

There is an example with `eth_sendTransaction` in `TestCallRPC`.

Reply:
Ah! I see